test_scripts/Resumption/InteriorVehicleData/011_InteriorVD_resumption_modules_after_unsubscription_unexpected_disconnect.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data after transport disconnect in case of
+--  processing unsubscription before resumption
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app with REMOTE_CONTROL hmi type is registered and activated
+-- 3. App is subscribed to module_1, module_2 and module_3
+-- 4. App is unsubscribed from module_2 and module_3
+-- 5. App receives updated hashId after unsubscription
+--
+-- Sequence:
+-- 1. Transport disconnect and reconnect are performed
+-- 2. App starts registration with actual hashId after unexpected disconnect
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(subscribe=true, module_1) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to mobile app
```

Comment:
b. SDL get a successful response for GetInteriorVehicleData request regarding moduleType_1 with isSubscribed=true
c. SDL respond RAI(success=true, result code = SUCCESS) to mobile app
d. update hashId after successful resumption

Reply:
Done